### PR TITLE
Repoclosures manifest

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
This might get cleaner if pkg_manifest branch is rebased. I am mostly opening this here to not lose track of it and it will require the full package_manifest.yml to be in the mainline code base.

File: repoclosure/yum_f27.conf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# repos
+[f27-base]
+name=BaseOS
+enabled=1
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=fedora-27&arch=x86_64
+failovermethod=priority
+
+[f27-puppet-5]
+name=f27-puppet-5
+baseurl=http://yum.puppetlabs.com/puppet5/fedora/27/$basearch/
```
Comment (ekohl) [inline repoclosure/yum_f27.conf:22]: hardcoded arch and variable mixing looks odd

Comment (ehelms) [reply repoclosure/yum_f27.conf:22]: You'd prefer using variables where possible? I can work that in.

**Comment by ehelms** [comment]
As I wrote the configs out, one thing I wondered was whether we should drop the OS identifier as that might enable more re-usability?

For example:

puppet-5 instead of el7-puppet-5 and let the repoclosure configuration file used drive the puppet-5 repository? This is less explicit in a way but potentially more re-usable.

**Pull request closed without merging**